1-org/envs/shared/log_sinks.tf

```diff
@@ -79,7 +79,7 @@ module "logs_export" {
     logging_sink_filter        = local.logs_filter
     log_bucket_id              = "AggregatedLogs"
     log_bucket_description     = "Project destination log bucket for aggregated logs"
-    location                   = local.default_region
+    location                   = try(var.log_export_storage_location, local.default_region)
```

Comment:
does this one also need to be replaced with `coalesce`

Reply:
Good catch! Fixed in 27d3c13efcdebdf5dcffe993baa8a5d5bdc914af